corehq/motech/models.py

```diff
@@ -286,9 +286,9 @@ def log(level: int, log_entry: RequestLogEntry):
             request_url=log_entry.url,
             request_headers=log_entry.headers,
             request_params=log_entry.params,
-            request_body=as_json_text(log_entry.data),
+            request_body=as_json_text(log_entry.data)[:MAX_REQUEST_LOG_LENGTH],
```

Comment:
this won't affect retries will it?

Reply:
No. Retries will fetch the form/case again (and rebuild the request body from its state at the time of the retry, not its state at the time of the previous attempt).